src/CameraSensorUtil.hh

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef GZ_SENSORS_CAMERASENSORUTIL_HH_
+#define GZ_SENSORS_CAMERASENSORUTIL_HH_
+
+#include <gz/math/Matrix4.hh>
+
+#include "gz/sensors/config.hh"
+#include "gz/sensors/Export.hh"
+
+namespace gz
+{
+namespace sensors
+{
+// Inline bracket to help doxygen filtering.
+inline namespace GZ_SENSORS_VERSION_NAMESPACE {
+/// \brief Computes the OpenGL NDC matrix
+/// \param[in] _left Left vertical clipping plane
+/// \param[in] _right Right vertical clipping plane
+/// \param[in] _bottom Bottom horizontal clipping plane
+/// \param[in] _top Top horizontal clipping plane
+/// \param[in] _near Distance to the nearer depth clipping plane
+///            This value is negative if the plane is to be behind
+///            the camera
+/// \param[in] _far Distance to the farther depth clipping plane
+///            This value is negative if the plane is to be behind
+///            the camera
+/// \return OpenGL NDC (Normalized Device Coordinates) matrix
+math::Matrix4d BuildNDCMatrix(double _left, double _right, double _bottom,
```

Comment:
for these global functions, gazebo uses lowercase first letter in their names, e.g. like the functions in [Utils.hh](https://github.com/gazebosim/gz-sensors/blob/gz-sensors7/include/gz/sensors/Util.hh). To be consistent, can you change this to `buildNDCMatrix`? Same for the other function names in this file.

Reply:
Done.